src/publisher/tests/test_ejp_article_ingest.py

```diff
@@ -66,40 +66,84 @@ def test_ejp_ingest_data(self):
 
     def test_ejp_ingest_over_existing_data_with_defaults(self):
         "importing ejp articles over existing articles causes an error"
```

Comment:
"importing EJP stubs over existing articles isn't possible unless `update` is explicitly set to `True`"